### ISSUE TITLE
Is the weight of BSR trained through v-prediction? 

### ISSUE THREAD

**Issue by xuhang07** [EternalEvan/FlowIE#17]
I have encountered some confusion in the inference code for the BSR task. Specifically, the details are as follows:

        latents = forward_flowie_one_step(model, init_noise,y_null,c = torch.cat([c_latent], 1))
        latents = latents.detach() / vae.config.scaling_factor
        img_buffer = vae.decode(latents).sample/2+0.5

Here, the latent is obtained through forward_flowie_one_step:

         ts = torch.ones(latents.shape[0]).to(latents)*timestep
         control = model.control_model(
                    latents, hint=c,
                    timesteps=ts, context=prompt_embeds
                        )
         noise_pred = model.model.diffusion_model(latents, timesteps=ts, context=prompt_embeds,control = 
                     control,only_mid_control=model.only_mid_control)
         return noise_pred

Here, the latent is essentially the noise_pred, which is directly inferred by the diffusion model. This raises some confusion, as it seems the weights here are obtained through x-prediction. Additionally, Mean Value Sampling appears to be implemented only in the BFR task.
We kindly request the author's clarification on this matter.
